homepage.html

```diff
@@ -0,0 +1,73 @@
+                    <a href="portfoliopage.html">Portfolio</a>
+                    <a href="resumepage.html">Resume</a>
+                </nav>
+            </h1>
+        </header>
+        <section>
+            <a href="resumepage.html"><img class="headshot" src="https://image.ibb.co/d6Qkoc/headshot1.png"></a>
+        </section>
+        <article>
+            <h1>About Me</h1>
+            <h2>A Short Story</h2>
+            <p>My name is Jacob Heivly. I am a 24 year old, aspiring software developer from 
+                Chapel Hill, North Carolina. I grew up as a hyperactive child constantly wanting to try something
+                new. Pretty soon I found a passion for skateboarding. I was able to enjoy this activity by my rules.
+                Skateboarding taught me to never give up. We beat ourselves up for hours to achieve goals. Try, adjust, try again...
+                (sounds a lot like coding, right?). After a decade of skateboarding I packed up my belongings to move west into
+                the mecca of the American skate scene, Southern California. Living solo in California was a reality check
+                to say the least. It's an incredibly competitive economy and the cost of living is out of this world 
+                compared to North Carolina. I learned to cut costs by any means which involved: minimizing possessions, living in a van,
+                living on a boat, and taking any work opportunity. Four years had past and I had found myself a decent
+                job managing at a skate shop in historical Carlsbad, CA. Life was better than it had been, but I was
+                still working paycheck to paycheck and hardly skateboarding, the passion that had brought me to San Diego.
+                It was at this point that I knew I needed to make a major change. 
+            </p>
+            <h2>Why did I choose Software Development?</h2>
+            <p>The technical world isn't going away any time soon. All modern companies can be considered 
+                "tech companies", and without developers there is no "tech". These skills are in high 
+                demand. My decade-long involvement with the skateboard community led me to believe that software
+                development would be a great fit for me. What does skateboarding have to do with coding? Skateboarders
+                are relentless. We will give every ounce of energy, fall over and over to accomplish a goal that we
+                set for ourselves. It's trial and error, just like development. 
+                <br>
+                <br>
+                Towards the end of my recent stay in San Diego, I found Momentum. A fully immersive, three month program 
+                led by industry professionals; and it was ten minutes away from where I grew up! I plan to acquire the 
+                skills to understand both front and back end development in preparation for jobs in this industy. Excitement would 
```

Comment:
This part doesn't show up on the screen -- need a tweak to your CSS to display this.